package.json

```diff
@@ -3,27 +3,67 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "firebase": "^9.15.0",
+    "firebase-tools": "^11.19.0",
+    "firestore": "^1.1.6",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-router-dom": "^6.5.0",
+    "typescript": "^4.9.4"
+  },
+  "scripts": {
+    "start": "vite",
+    "build": "tsc && vite build",
+    "lint": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
+    "preview": "vite preview",
+    "lint:fix": "eslint --fix --ext .js,.jsx,.tsx,.ts .",
+    "test:watch": "vitest watch",
+    "test:coverage": "vitest run --coverage",
+    "test": "vitest run"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "devDependencies": {
     "@patternfly/react-core": "^4.267.6",
```

Comment:
Are you sure this shouldn't be a regular dependency?

Reply:
deployed app of this pr works well but I'll move it to regular dependency